hex/spec/dependabot/hex/update_checker_spec.rb

```diff
@@ -532,6 +532,42 @@
 
       it { is_expected.to be >= Gem::Version.new("1.4.3") }
     end
+
+    context "with sub projects" do
+      let(:mixfile_body) { fixture("mixfiles", "sub_projects") }
+      let(:lockfile_body) { fixture("lockfiles", "sub_projects") }
+      let(:files) { [mixfile, lockfile, sub_mixfile1, sub_mixfile2] }
+      let(:sub_mixfile1) do
+        Dependabot::DependencyFile.new(
+          name: "apps/dependabot_business/mix.exs",
+          content: fixture("mixfiles", "dependabot_business")
+        )
+      end
+      let(:sub_mixfile2) do
+        Dependabot::DependencyFile.new(
+          name: "apps/dependabot_web/mix.exs",
+          content: fixture("mixfiles", "dependabot_web")
+        )
+      end
```

Comment:
Once you've got the project fixture set up this should work:
```suggestion
      let(:files) { project_dependency_files("umbrella_sub_projects") }
```